test/tiering_tests.py

```diff
@@ -64,14 +65,16 @@ def check_kmem_dax_support(self):
         else:
             return True
 
-    def get_ld_preload_cmd_output(self, tiers_config, thresholds_config=None,
-                                  log_level=None, negative_test=False):
+    def get_ld_preload_cmd_output(self, tiers_config, mtt_limits=None,
+                                  thresholds_config=None, log_level=None,
+                                  negative_test=False):
         log_level_env = self.log_prefix + "LOG_LEVEL=" + log_level \
             if log_level else ""
+        mtt_limits_env = self.mtt_limits_env_var + '="' + mtt_limits + '"'
```

Comment:
Why aren't we using the ```f"some_{var}"``` syntax ? Is it for legacy reasons ? It looks more readable